hibernate-reactive-core/src/test/java/org/hibernate/reactive/BaseReactiveTest.java

```diff
@@ -120,6 +120,14 @@ public void after(TestContext context) {
 		return sessionFactory.unwrap( Stage.SessionFactory.class );
 	}
 
+	protected Stage.Session getSession() {
```

Comment:
I think we should call this `createSession` or similar

Reply:
Also it's not used anywhere. do we want' to include it anyway for the future?

Reply:
Yes, it's there for people who want this convenience when writing new tests.